differential/diffAtlas_utils.R

```diff
@@ -181,12 +181,12 @@ make_exp_contrast_table <- function(analytics){
     formula
   })
   
-  return( conts )
```

Comment:
I wonder if it would be better to be explicit here. In the end, as we jump all day from R to bash, to python and to other things, explicit readability might help. But up to you.

Reply:
Probably this was done to harmonize the output of the R functions, i.e. either all use the `return()` explicitly or any.